shell/common/rasterizer.cc

```diff
@@ -399,9 +399,9 @@ static sk_sp<SkData> ScreenshotLayerTreeAsImage(
   SkMatrix root_surface_transformation;
   root_surface_transformation.reset();
 
-  auto frame = compositor_context.AcquireFrame(surface_context, canvas, nullptr,
-                                               root_surface_transformation,
-                                               false, nullptr);
+  auto frame = compositor_context.flutter::CompositorContext::AcquireFrame(
```

Comment:
It is not immediately clear why this would needs to be done.  Please document this in a comment and also note that this is only an issue on Fuchsia where the compositor context is different.